modules/oadp-1-4-0-release-notes.adoc

```diff
@@ -0,0 +1,66 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-06-28
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-1-4-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="oadp-1-4-0-release-notes_{context}"]
+= OADP 1.4.0 release notes
+
+The {oadp-first} 1.4.0 release notes lists resolved issues and known issues.
+
+[id="resolved-issues-1-4-0_{context}"]
+== Resolved issues
+
+.Restore works correctly in {product-title} 4.16
+
+Previously, while restoring the deleted application namespace, the restore operation partially failed with the `resource name may not be empty` error in {product-title} 4.16.
+With this update, restore works as expected in {product-title} 4.16.
+link:https://issues.redhat.com/browse/OADP-4075[OADP-4075]
+
+.Data Mover backups work properly in the {product-title} 4.16 cluster
+
+Previously, Velero was using the earlier version of SDK where the `Spec.SourceVolumeMode` field did not exist. As a consequence, Data Mover backups failed in the {product-title} 4.16 cluster on the external snapshotter with version 4.2.
+With this update, external snapshotter is upgraded to version 7.0 and later. As a result, backups do not fail in the {product-title} 4.16 cluster.
+link:https://issues.redhat.com/browse/OADP-3922[OADP-3922]
+
+For a complete list of all issues resolved in this release, see the list of link:https://issues.redhat.com/issues/?filter=12438505[OADP 1.4.0 resolved issues] in Jira.
+
+
+[id="known-issues-1-4-0_{context}"]
+== Known issues
+
+.Backup fails when checksumAlgorithm is not set for MCG
+
+While performing a backup of any application with Noobaa as the backup location, if the `checksumAlgorithm` configuration parameter is not set, backup fails. To fix this problem, if you do not provide a value for `checksumAlgorithm` in the Backup Storage Location (BSL) configuration, an empty value is added.
+The empty value is only added for BSLs that are created using Data Protection Application (DPA) custom resource (CR), and this value is not added if BSLs are created using any other method.
+link:https://issues.redhat.com/browse/OADP-4274[OADP-4274]
+
+For a complete list of all known issues in this release, see the list of link:https://issues.redhat.com/issues/?filter=12438506[OADP 1.4.0 known issues] in Jira.
+
+
+[id="upgrade-notes-1-4-0_{context}"]
+== Upgrade notes
+
+[NOTE]
```

Comment:
Since this seems like a required step, changing from a NOTE to IMPORTANT seems appropriate here: https://redhat-documentation.github.io/supplementary-style-guide/#admonitions